onnxruntime/python/tools/quantization/__init__.py

```diff
@@ -3,5 +3,5 @@
 from .calibrate import CalibrationDataReader, calculate_calibration_data, get_calibrator, generate_calibration_table
 from .calibrate import calibrate
 from .quant_utils import QuantType, write_calibration_table
-from .evaluate import YoloV3Evaluator, YoloV3VisionEvaluator
-from .data_reader import YoloV3DataReader, YoloV3VisionDataReader
+from .evaluate import YoloV3Evaluator, YoloV3VisionEvaluator, TRTResNet50Evaluator
```

Comment:
upon further reflection, i feel like we need to generalize this better.
We shouldn't have things like YoloV3 , TRTResnet evaluators. 
Let's try to be more generic. it won't scale if we need to have specific implementations for every new model/accelerator we add here.